README.md

```diff
@@ -60,24 +79,45 @@ Then, add the package(s):
   ];
 }
 ```
-
 Everything is available as an overlay if you prefer that.
 
-### Overrides for osu
+**NOTE**: currently using Home Manager with flakes will not evaluate the package.
+For that reason it's recommended to use the overlay.
 
-The osu derivation was written with versatility in mind. There are args that can be modified in order to get the result one wants.
+### Nix Stable
+
+The following instructions assume you have this repo cloned somewhere.
+
+#### Packages
+
+To install packages with `nix-env`, run
+```
+cd path/to/osu.nix
```

Comment:
Should do something like this instead to be more precise.
```suggestion
cd directory/of/osu.nix
```

Reply:
Good suggestion, thanks!